.github/workflows/check-examples.yml

```diff
@@ -42,12 +42,14 @@ jobs:
         python-version: 3.9
     - name: install python requirements for notebooks
       run: |
+        cd pytket-examples-book
         python -m pip install --upgrade pip
         python -m pip install pre-commit==3.0.0
         python -m pip install wheel
-        python -m pip install -r requirements.txt
-        python -m pip install -c constraints.txt jupyter openfermion plotly
-        python -m pip install docker p2j pytest
+        python -m pip install -r example_requirements.txt
+        python -m pip install pytket-qsharp == 0.34.0
```

Comment:
I have added the qsharp version here instead of the `example_requirements.txt` as I want to maintain the ability to built the jupyterbook locally on my M1 mac. The qsharp installation causes issues locally.